lib/ethereumex/smart_contracts/deploy.ex

```diff
@@ -0,0 +1,71 @@
+defmodule Ethereumex.SmartContracts.Deploy do
+  @moduledoc false
+
+  alias Ethereumex.HttpClient
+
+  defmodule Error do
+    defexception message: nil, value: nil
+  end
+
+  @spec execute(binary, keyword) :: binary
+  def execute(compiled_contract, opts \\ []) do
+    from = opts[:from] |> prepare_from_value
+    gas = opts[:gas] |> prepare_gas_value(compiled_contract, from)
+
+    send_transaction(compiled_contract, from, gas)
+  end
+
+  @spec send_transaction(binary, binary, binary) :: binary
+  defp send_transaction(compiled_contract, from, gas) do
+    response =
+      [%{from: from, gas: gas, data: compiled_contract}]
```

Comment:
What do you think about the `ShortMaps` Lib to reduce this:
```
[%{from: from, gas: gas, data: compiled_contract}]
```
to this
```
[~M{from, gas, data: compiled_contract}]
```

Reply:
thanks for suggestion, @eduardonunesp. This ShorterMaps library looks interesting. But we should keep
external dependencies number as small as possible.

Reply:
:+1: that's make sense